source/material_model/rheology/elasticity.cc

```diff
@@ -127,10 +129,9 @@ namespace aspect
           AssertThrow(use_fixed_elastic_time_step == true,
                       ExcMessage("Stress averaging can only be used if 'Use fixed elastic time step' is set to true'"));
 
-        stabilization_scale_factor = prm.get_double ("Stabilization scale factor");
-        if (stabilization_scale_factor < 1)
-          AssertThrow(stabilization_scale_factor < 1,
-                      ExcMessage("Stabilization scale factor must be greater than one"));
+        stabilization_time_scale_factor = prm.get_double ("Stabilization time scale factor");
+        AssertThrow(stabilization_time_scale_factor >= 1,
```

Comment:
Please make the following minor change to signal this is a `double` not an integer.

```suggestion
        AssertThrow(stabilization_time_scale_factor >= 1.,
```

Reply:
This assert is not necessary as the get_double should do the check, but I guess it doesn't hurt.